extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -381,21 +383,21 @@ export class AmpSidebar extends AMP.BaseElement {
       this.scheduleResume(children);
       tryFocus(this.element);
       this.triggerEvent_(SidebarEvents.OPEN);
+      this.element.setAttribute('aria-hidden', 'false');
     } else {
       // On close sidebar
       this.vsync_.mutate(() => {
         toggle(this.element, /* display */false);
         this.schedulePause(this.getRealChildren());
```

Comment:
Not sure if we're ok with the use of `scrollY`, but all of the scrolls are banned. I think this should be fine because it's in a `vsync_.mutate` block, but please lmk if there's a better way.

Reply:
I think one big problem with `window.scrollY` is that it doesn't work correctly in the viewer. You might want to use viewport service methods instead, like [`Viewport#getScrollTop`](https://github.com/ampproject/amphtml/blob/master/src/service/viewport/viewport-impl.js#L272)

Reply:
I refactored this to use Viewport#getScrollTop, but this doesn't seem to work all the time. For example, in the test that is currently breaking, we can observe a change in `win.scrollY`, but not a change in `viewport_.getScrollTop`. The test is basically a simulation of a manual test situation of clicking on a `idOfSection.scrollTo` and then verifying that we don't try to re-scroll and refocus since the scroll changed while the sidebar was open. What is special about our test setup that causes `win.scrollY` to accurately reflect changes but `viewport.getScrollTop` to not?